site/assets/css/fish_style.css

```diff
@@ -263,9 +263,7 @@ h3 {
     image-rendering: -o-crisp-edges;
     image-rendering: -webkit-optimize-contrast;
     image-rendering: -webkit-crisp-edges;
-    image-rendering: pixelated;
-    image-rendering: optimize-contrast; 
-    image-rendering: crisp-edges;
+    image-rendering: optimize-contrast;
```

Comment:
It's definitely wrong if it says that's true everywhere. You will want to evaluate what it is reporting to you for something like this just like you would in the Safari/Chrome inspectors - from the perspective of each browser that can't know why we do this, form its perspective a few of those rules were not necessary. I pull up the web inspector in Safari or Chrome for example, it will report some unnecessary properties there - unnecessary and overwritten here but that is not the case on an older version of Safari or any Mozilla (or IE). You will see browser-specific fix-ups that make the browsers they are not needed for unhappy every day in web development.